gc/base/GlobalCollector.hpp

```diff
@@ -102,6 +102,9 @@ class MM_GlobalCollector : public MM_Collector {
 
 	virtual void checkColorAndMark(MM_EnvironmentBase* env, omrobjectptr_t objectPt) { Assert_MM_unreachable(); }
 
+	/* Size in bytes to be reserved in an allocation cache for the collector. Required for specific use cases (SATB TLH Premark) */
+	virtual uintptr_t reserveAllocCacheSize() { return 0; }
```

Comment:
you can add with By/ForGC (if you want the method name to finish with 'Size', it can be in a middle: reservedForGCAllocCacheSize(),
again just to be a bit more clear for whom it's reserved